main.tf

```
@@ -1,13 +1,69 @@
-resource "random_integer" "example" {
-  count = module.this.enabled ? 1 : 0
+locals {
+  enabled      = module.this.enabled
+  cluster_name = join("", aws_ecs_cluster.default.*.name)
+
+  capacity_providers_fargate = [for name, is_enabled in {
+    FARGATE : var.capacity_providers_fargate,
+    FARGATE_SPOT : var.capacity_providers_fargate_spot
+  } : name if is_enabled]
```

This is more readable

```suggestion
  capacity_providers_fargate = [
    for name, is_enabled in {
      FARGATE      = var.capacity_providers_fargate,
      FARGATE_SPOT = var.capacity_providers_fargate_spot
    } : name if is_enabled
  ]
```